frontend/coreproject/src/lib/components/pages/anime_info/SidebarDetails.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+	export let anime: any;
+
+	// icons
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import TrendingUp from '$icons/Trending-Up.svelte';
+	import Star from '$icons/Star.svelte';
+	import Edit from '$icons/Edit.svelte';
+	import ExternalLink from '$icons/ExternalLink.svelte';
+</script>
+
+<div>
+	<div class="flex items-center gap-4">
+		<h4 class="font-semibold 2xl:!text-2xl 3xl:!text-3xl">Ratings</h4>
+		<SettingsOutline width="15" height="15" class="opacity-75" />
+	</div>
+
+	<div class="xl:mt-4 2xl:mt-5">
+		<div>
+			<span class="xl:text-3xl 2xl:text-4xl 3xl:text-5xl font-bold underline decoration-white/25 underline-offset-[15px]"
+				>{Math.ceil((anime.rating / 5) * 100)}%</span
+			>
+			<span class="xl:text-xs 2xl:text-base 3xl:text-lg">| {anime.totalResponse} Ratings</span>
```

Comment:
`|` is not a good thing

https://www.skeleton.dev/elements/core#vertical-dividers